### ISSUE TITLE
Adding support for pixelBuffer input for CoreML models

### ISSUE THREAD

**Issue by fabiotu** [natmlx/natml#40]
For a model accepting byte image pixelBuffer:

```
MLEdgeModel
Input: input: (1, 256, 256, 4) System.Byte
Output: activation_out: (1, 256, 256, 4) System.Byte
```

At the moment model.Predict() fails with:

`Unity[2451:70453] NatML Error: Failed to make prediction with error: Error Domain=com.apple.CoreML Code=0 "The model expects input feature input to be an image, but the input (MultiArray : Float32 1 × 256 × 256 × 4 array) is of type 5." UserInfo={NSLocalizedDescription=The model expects input feature input to be an image, but the input (MultiArray : Float32 1 × 256 × 256 × 4 array) is of type 5.}`

when submitting an MLArrayFeature containing raw pixels.

To test you can use, for example: https://github.com/john-rocky/CoreML-Models#photo2cartoon

**Comment by olokobayusuf** [comment]
We'll get to this soon, thanks for bringing it up!

**Comment by olokobayusuf** [comment]
This has been implemented in [NatML 1.1.2](https://github.com/natmlx/natml/releases/tag/1.1.2).
